File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // Gets height until in range 1 to 8
    do
    {
        // Gets height
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);



    // To make pyramid
    int space = height;

    for (int i = 0; i < height; i++)
    {
        // Prints spaces
        for (int j = 1; j < space; j++)
        {
            printf(" ");
        }
        space--;

        int hashes = height - space;

        // Prints #s
        for (int k = 0; k < hashes; k++)
        {
            printf("#");
        }

        // Print the middle spaces
        printf("  ");

        // Print second half of hashes
        for (int k = 0; k < hashes; k++)
        {
            printf("#");
        }
        // Go to a new line
        printf("\n");



    }


}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
# Get user input
height = 0
while (height < 1 or height > 8):
    height = get_int("Height:")

row = 1
# Iterate through rows
for i in range(height):
    # Print spaces then hashes
    for j in range(height - row):
        print(" ", end="")

    for k in range(row):
        print("#", end="")

    # Print middle space
    print("  ", end="")

    # Print end hashes
    for k in range(row):
        print("#", end="")
    # Go to the next line
    print("")
    row += 1